code/modules/projectiles/guns/ballistic/shotgun.dm

```diff
@@ -192,6 +192,21 @@
 	var/slung = FALSE
 	can_bayonet = TRUE //STOP WATCHING THIS FILTH MY FELLOW CARGONIAN,WE MUST DEFEND OURSELVES
 
+/obj/item/gun/ballistic/shotgun/doublebarrel/knuckleblasterinhand
```

Comment:
you say that this is an improvised shotgun but it's not a subtype of the improvised one

Reply:
didnt say it was improvised, it just has the mag of one

Reply:
That's a relief, I'd hate to spend 10tc on a sneaky shotgun that sometimes jams